recipe/meta.yaml

```
@@ -8,19 +8,21 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 3245114253b6adb4b38f2a2a1828edfcd9e8c012f7e30a5cef1932ca7344eb44
+  sha256: 42e399a66c8150dc507602dff7b7953f105ef11faf97ddaa6d27b1cbf45c4c98
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: True # [py<37]
 
 requirements:
   host:
     - pip
-    - python >=3.7,<4
+    - python
+    - setuptools
+    - wheel
   run:
-    - python >=3.7,<4
+    - python
     - about-time ==4.2.1
     - grapheme ==0.6.0
```

You can eventually relax the pinnings here:

```suggestion
    - about-time >=4.2.1
    - grapheme >=0.6.0
```

---

https://github.com/rsalmei/alive-progress/blob/main/setup.py#L60

The versions are pinned here, we should keep it pinned for the time being unless they change this

---

@JamesRobertsonGames @psteyer https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/02_Recipes/Pinnings.md#dont-necessarily-follow-what-upstream-does-to-the-letter